public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,115 @@
+# Setup a Google Drive DVC Remote
+
+Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
+[remote storage](/doc/command-reference/remote).
+
+## Configure a Google Cloud _project_
```

Comment:
let's remove italic here for now, looks like right panel does not handle this well:
<img width="917" alt="Screen Shot 2020-01-24 at 4 31 15 PM" src="https://user-images.githubusercontent.com/3659196/73113199-5f822480-3ec7-11ea-90c8-8897d0664948.png">